test/scraper/schleswigholstein_landtag_scraper_overview_test.rb

```diff
@@ -0,0 +1,70 @@
+require 'test_helper'
+
+class SchleswigholsteinLandtagScraperOverviewTest < ActiveSupport::TestCase
+  def setup
+    @scraper = SchleswigHolsteinLandtagScraper
+    @html = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/schleswigholstein_landtag_scraper_overview.html')
+                           ).force_encoding("WINDOWS-1252"))
+    @table = @scraper.extract_table @html
+    @blocks = @scraper.extract_blocks @table
+  end
+
+  test 'extract table from resultpage' do
+    assert_equal 2679, @table.children.length
+  end
+
+  test 'get blocks from table' do
+    @table = @scraper.extract_table @html
+    assert_equal 1338, @blocks.length
+  end
+
+  test 'get title from block' do
+    assert_equal "Kinderschutz durch Therapie pädophil veranlagter Menschen", @scraper.extract_title(@blocks[0])
+  end
+
+  test 'is answered' do
+    assert_equal true, @scraper.answer?(@blocks[0])
+    assert_equal false, @scraper.answer?(@blocks[1])
+  end
+
+  test 'get ministries' do
+    meta = @scraper.extract_meta(@blocks[0])
+    assert_equal ["Minister/in für Soziales, Gesundheit, Wissenschaft und Gleichstellung"], meta[:ministries]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.